### PR TITLE
Placeholder for comments.

### PR DESCRIPTION
**Pull request by aprotim**

### CODE REVIEW

File: content.md [hunk 1]
```diff
@@ -1,125 +1,126 @@
...
+# Aprotim Sanyal 
+ 
+jobs@aprotim.com   
+(310) 243-6230 
+ 
+Employment 
+------------ 
+### Edgecast - **Principal Software Engineer, SRE** - *April 2020 - July 2022* 
+ 
+Rather than focusing on a single service, as a member of a small SRE task force for the entire organization, I identified areas of cross-cutting concern where we could act as a catalyst to amplify effort across other engineering teams and led several of these efforts.
```
Comment (Marasco77) [inline content.md]: As a Principal SRE on an organization-wide task force, I worked cross-functionally to identify company-wide areas of concern and implement standardized practices to reduce manual work while improving system stability and reliability.

File: content.md [hunk 2]
```diff
@@ -1,125 +1,126 @@
...
+jobs@aprotim.com   
+(310) 243-6230 
+ 
+Employment 
+------------ 
+### Edgecast - **Principal Software Engineer, SRE** - *April 2020 - July 2022* 
+ 
+Rather than focusing on a single service, as a member of a small SRE task force for the entire organization, I identified areas of cross-cutting concern where we could act as a catalyst to amplify effort across other engineering teams and led several of these efforts. 
+ 
+- **Built organization-wide best practices.** My primary responsibility as an experienced SRE was evaluating existing procedures and tools
```
Comment (Marasco77) [inline content.md]: - **Built organization-wide best practices.** Evaluated existing processes and tooling across EdgeCast, recommending and implementing changes in line with industry standards

File: content.md [hunk 3]
```diff
@@ -1,125 +1,126 @@
...
+Employment 
+------------ 
+### Edgecast - **Principal Software Engineer, SRE** - *April 2020 - July 2022* 
+ 
+Rather than focusing on a single service, as a member of a small SRE task force for the entire organization, I identified areas of cross-cutting concern where we could act as a catalyst to amplify effort across other engineering teams and led several of these efforts. 
+ 
+- **Built organization-wide best practices.** My primary responsibility as an experienced SRE was evaluating existing procedures and tools 
+  across EdgeCast to create consistent processes and tooling in line with best practices. 
+  - **Evangelized SRE best practices to leadership.** Educated senior directors and VPs on the value of **SLIs/SLOs, error budgets, 
+    consistent tooling and practices**, etc. to build a better data-driven platform.
```
Comment (Marasco77) [inline content.md]: Consistent tooling and practices, and other SRE principles to build a data-driven platform for improved production reliability and strategic decision-making

File: content.md [hunk 4]
```diff
@@ -1,125 +1,126 @@
...
+### Edgecast - **Principal Software Engineer, SRE** - *April 2020 - July 2022* 
+ 
+Rather than focusing on a single service, as a member of a small SRE task force for the entire organization, I identified areas of cross-cutting concern where we could act as a catalyst to amplify effort across other engineering teams and led several of these efforts. 
+ 
+- **Built organization-wide best practices.** My primary responsibility as an experienced SRE was evaluating existing procedures and tools 
+  across EdgeCast to create consistent processes and tooling in line with best practices. 
+  - **Evangelized SRE best practices to leadership.** Educated senior directors and VPs on the value of **SLIs/SLOs, error budgets, 
+    consistent tooling and practices**, etc. to build a better data-driven platform. 
+  - **Established SLIs and SLOs across teams.** Having established the value of SL\[IO\]s, I took 
+    on the large task of meeting with service teams across the org to understand their services and dependencies. I explained what makes useful, actionable SLIs, and coached team representatives through writing and instrumenting new SLIs and SLOs in line with these standards.
```
Comment (Marasco77) [inline content.md]: Led company-wide initiative with service teams to understand responsibilities and dependencies, and provide training to cross-functional engineering teams on establishing actionable SLIs. Extensively coached teams on writing and instrumenting new SLIs and SLOs in line with improved standards

File: content.md [hunk 5]
```diff
@@ -1,125 +1,126 @@
...
+    on the large task of meeting with service teams across the org to understand their services and dependencies. I explained what makes useful, actionable SLIs, and coached team representatives through writing and instrumenting new SLIs and SLOs in line with these standards. 
+    This enabled the platform as a whole to move away from a reactive, incident-driven maintenance model towards proactive, error-budget-driven development. 
+  - **Created and documented escalation best practices for oncall engineers.** 
+    - **Establiblished standard oncall response SLAs and expectations.** After noticing differing and undocumented oncall response across teams different teams, I took on the task of creating common expectations across teams. I empaneled meetings with various stakeholders (NOC, service owners, leadership) to understand each team's implicit expectations for escalations (both their own and other teams'). Based on this input, I set up example service tiers with **specific SLAs around response time**, **definitions** for what "response" entailed, and **standards** for how to publish each team's oncall SLA and escalation procedures where they could be seen by others.
```
Comment (Marasco77) [inline content.md]: Identified operational deficiencies stemming from lack of organizational standards for oncall response. Worked cross-functionally with stakeholders (NOC, service owners, leadership) to identify and document expectations for response time and escalation procedures. Based on this input, I designed example service tiers with **specific SLAs around response time**, **definitions** for what "response" entailed, and **standards** for how to publish each team's oncall SLA and escalation procedures where they could be seen by others.

File: content.md [hunk 6]
```diff
@@ -1,125 +1,126 @@
...
+    This enabled the platform as a whole to move away from a reactive, incident-driven maintenance model towards proactive, error-budget-driven development. 
+  - **Created and documented escalation best practices for oncall engineers.** 
+    - **Establiblished standard oncall response SLAs and expectations.** After noticing differing and undocumented oncall response across teams different teams, I took on the task of creating common expectations across teams. I empaneled meetings with various stakeholders (NOC, service owners, leadership) to understand each team's implicit expectations for escalations (both their own and other teams'). Based on this input, I set up example service tiers with **specific SLAs around response time**, **definitions** for what "response" entailed, and **standards** for how to publish each team's oncall SLA and escalation procedures where they could be seen by others. 
+    - When onboarding the main CDN dev team onto the a shared rotation with SREs, I documented oncall expectations in line with the standards I created above.
```
Comment (Marasco77) [inline content.md]: Onboarded CDN dev team to shared SRE rotation, documenting expectations and process in line with newly established standards, providing an example process to the organization

File: content.md [hunk 7]
```diff
@@ -1,125 +1,126 @@
...
+    This enabled the platform as a whole to move away from a reactive, incident-driven maintenance model towards proactive, error-budget-driven development. 
+  - **Created and documented escalation best practices for oncall engineers.** 
+    - **Establiblished standard oncall response SLAs and expectations.** After noticing differing and undocumented oncall response across teams different teams, I took on the task of creating common expectations across teams. I empaneled meetings with various stakeholders (NOC, service owners, leadership) to understand each team's implicit expectations for escalations (both their own and other teams'). Based on this input, I set up example service tiers with **specific SLAs around response time**, **definitions** for what "response" entailed, and **standards** for how to publish each team's oncall SLA and escalation procedures where they could be seen by others. 
+    - When onboarding the main CDN dev team onto the a shared rotation with SREs, I documented oncall expectations in line with the standards I created above. 
+- **Provided technical solutions to reliability problems.**
```
Comment (Marasco77) [inline content.md]: Provided is weak, maybe Architected?

File: content.md [hunk 8]
```diff
@@ -1,125 +1,126 @@
...
+  - Built on **Google Cloud Platform** to manage provisioning of storage, storage class migration, data retention, authentication/authorization, key escrow, and encryption. 
+  - Made setup and restore **self-service**, leading to happier users and less toil on the team. 
+- **Set technical direction** for a team of between **2 and 6 engineers** to build, test, and roll out our new service. 
+- **Managed coordination with other teams** to: 
+  - **Push for upstream features** in Google Cloud Storage by speccing requirements, coordinating changes, and finding headcount from internal initiatives. 
+  - **Capture user requirements** by working closely with alpha-/beta-testers to capture pain points, hard requirements, and wishlist features. 
+  - **Conform to security best practices**, seeking input from security team and building out features as needed to help users avoid security pitfalls. 
+ 
+#### Enterprise Data Storage - Individual Contributor - *(2014 - Present)* 
+- **Migrated XX,000 long-term storage images** from deprecated tape system onto cloud storage, including integrity checking and modifying our backups management software.
```
Comment (Marasco77) [inline content.md]: Is XX intentional or a placeholder?

File: content.md [hunk 9]
```diff
@@ -1,125 +1,126 @@
...
+ 
+Volunteering/Service 
+-------------------- 
+    
+### CSSI@Google LAX 
+ 
+[CSSI](https://buildyourfuture.withgoogle.com/programs/computer-science-summer-institute/) is a three-week intensive programming course for rising college freshman, primarily from groups underrepresented in tech. It culminates in students designing, implementing, and building their own web app. 
+ 
+#### Co-Lead Instructor - Summer 2018 and 2019 
+- **Ran Google L.A.'s CSSI instruction** for two summers, along with my co-lead.
```
Comment (Marasco77) [inline content.md]: I think you can strike “along with co-lead”

File: content.md (hunk 8 above)
Comment (aprotim) [reply content.md]: Intentional for NDA reasons. (Also I now do not remember the number.) I can say "tens of thousands" to make it clearer?

File: content.md [hunk 10]
```diff
@@ -1,125 +1,126 @@
...
+    - When onboarding the main CDN dev team onto the a shared rotation with SREs, I documented oncall expectations in line with the standards I created above. 
+- **Provided technical solutions to reliability problems.** 
+  - Moved configuration integration testing into its own service, independent from code testing, resulting in a more stable and consistent build/certification pipeline. 
+  - **Designed solution for a major reliability problem** in which a single node failures caused delays and buffering across all videos and large files delivered. Worked with the CDN Devs to implement this solution to identify and route around failed nodes even before they were removed from the working pool. 
+ 
+ 
+### Google, Inc. - **Site Reliability Engineer** - *February 2008 - February 2020* 
+ 
+During my tenure as an SRE, I've built and run Google enterprise infrastructure using open source tools on vanilla-ish Linux,
```
Comment (Marasco77) [inline content.md]: During my tenure as an SRE, I built and ran Google enterprise infrastructure on a range of platforms including using open source tools on Linux, Google’s production infrastructure, and Google Cloud Platform. Provided oncall support with 5- and 30-minute response SLAs, and continously evaluated issues and responses and built tooling and processes to streamline operations and automate away toil

File: content.md (hunk 9 above)
Comment (aprotim) [reply content.md]: My worry is, world being small and tech being incestuous, I don't want Michael to think I'm taking credit for his work, and also don't want anybody who knows Michaels CSSI work to be like, "this guy's lying".  Removed the "along with..." and changed the verb to Co-Ran.

**Pull request merged**